autopush/router/webpush.py

```diff
@@ -47,10 +47,11 @@ def _crypto_headers(self, notification):
             encoding=headers["content-encoding"],
             encryption=headers["encryption"],
         )
-        # AWS cannot store empty strings, so we only add the encryption-key if
-        # its present to avoid empty strings.
-        if "encryption-key" in headers:
-            data["encryption_key"] = headers["encryption-key"]
+        # AWS cannot store empty strings, so we only add these keys if
+        # they're present to avoid empty strings.
+        for name in ["encryption-key", "crypto-key"]:
+            if name in headers:
+                data[name] = headers[name]
```

Comment:
The - to _ is more for logs than for data exchange. These values need to stay consistent between routing and eventual delivery to clients. Changing these values here could break dependency outside of the app. 

Still, nice spotting. Thanks!